### DISCUSSION TITLE
Welcome to Data-analysis Discussions!

### DISCUSSION THREAD

**Discussion by Yaa89ps** [Yaa89ps/Data-analysis#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Yaa89ps** [comment]
Coucou Louise !!!!

**Reply by louisehelary** [reply]
coucou !!!

**Comment by Yaa89ps** [comment]
Le Github du gars qui a fait la carte interactive !

https://github.com/Guillaume-Fgt/Folium_Villes_Fleuries_Idf
